pkg/core/transactor/collector.go

```diff
@@ -0,0 +1,38 @@
+package transactor
+
+import (
+	"bytes"
+	"encoding/binary"
+
+	"github.com/dusk-network/dusk-blockchain/pkg/p2p/wire"
+	"github.com/dusk-network/dusk-blockchain/pkg/p2p/wire/encoding"
+	"github.com/dusk-network/dusk-blockchain/pkg/p2p/wire/topics"
+)
+
+type (
+	transferCollector struct {
+		transferChan chan transferInfo
+	}
+
+	transferInfo struct {
+		amount  uint64
+		address string
+	}
+)
+
+func initTransferCollector(subscriber wire.EventSubscriber) chan transferInfo {
```

Comment:
Is `initTransferCollector` used anywhere, or it's here only as an example?

Reply:
No, you're right. This is some leftover code that I forgot to remove, I modeled the transactor to work with channels at an earlier stage.